README.md

```diff
@@ -66,8 +66,17 @@ When you have implemented the tool, answer the questions below, commit it to Git
 
 How does your method for extracting features work?
 
+A table is created from the bedfile, this table includes start, stop and features, sorted by chromosome. in the query we iterate through the queried chromosome and range and append features to the file in a BED format
+
+
 What is the complexity of the algorithm, as a function of the size of the two input files? When you answer this, you need to know that you can get the list of chromosomse from a `query.Table` in constant time, but it does, of course, take longer to run through all the lines in it.
 
+The algorithm is a function of the two input files and all else being constant time, thus it is O(n*m)
```

Comment:
That is a little loose analysis. I can imagine many algorithms that have two input files of length $n$ and $m$ that wouldn't run in $O(nm)$... Why, for example, do you multiply $n$ and $m$ instead of adding them? Or why isn't it $O(n^2m^3)$?